build.gradle

```diff
@@ -538,6 +538,8 @@ project(':core') {
     duplicatesStrategy 'exclude'
   }
 
+  systemTestLibs.dependsOn('jar', 'copyDependantTestLibs')
```

Comment:
Note that in line 440, we also have a `systemTestLibs` (this was added after your PR was initially created). I think we need to modify that line instead of introducing this here.